src/pages/index.js

```diff
@@ -53,15 +53,7 @@ class Home extends Component {
           title="React &ndash; A JavaScript library for building user interfaces"
           canonicalUrl={createCanonicalUrl('/')}
         />
-<<<<<<< HEAD
-        <div dir="rtl"
```

Comment:
I think we should keep `dir="rtl"`

Reply:
I am going to create another ticket to update layout where these attributes are set in a better way and also those sets the correction direction from both HTML and CSS perspective.

Reply:
@armujahid here we are - https://github.com/reactjs/ur.reactjs.org/issues/25